src/_includes/cloud/note-docker-ssl-not-private-connection.md

```diff
@@ -0,0 +1,3 @@
+{:.bs-callout-info}
+If you see `Your connection is not private` error message while using `https` connection press `Advanced` button and then `Proceed to magento2.docker (unsafe)` link.
+In case if you are using Google Chrome and there is no `Advanced` button just simply type `thisisunsafe` 
```

Comment:
I suggest the following, which formats UI elements properly and finishes the sentence about Chrome's bypass. Are these suggestions still true to your intention? Maybe it will fix the listing errors, too.

```suggestion
If you see the `Your connection is not private` error message while using an HTTPS connection, click **Advanced**, then click the **Proceed to magento2.docker (unsafe)** link. If you use Google Chrome and there is no **Advanced** button, then type `thisisunsafe` to bypass the security warnings.
```